.github/workflows/test.yml

```diff
@@ -7,6 +7,24 @@ on:
     branches: ['**']
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      is_specific: ${{ steps.path-check.outputs.specific_dir }}
+      is_others: ${{ steps.path-check.outputs.others }}
+    steps:
+      - id: path-check
+        uses: dorny/paths-filter@v2
+        with:
+          list-files: shell
+          filters: |
+            specific_dir:
```

Comment:
Let’s rename this to something like „apple_dirs” in a follow-up PR and adjust the name of the job. „Changes” is super ambiguous.